import-dashboards.py

```diff
@@ -529,7 +530,88 @@ def set_home_dashboard(api_key):
     #             "SELECT 1, 1, 0, 0, id, '', '', datetime('now'), datetime('now') from dashboard WHERE slug='home'")
 
 
+def dbaas_dashboard(api_key):
+    if DBAAS is None or DBAAS.lower() in ('0','false','f'):
+        print ' * DBaaS is disabled'
+        r = requests.get('%s/api/dashboards/uid/pmm-dbaas' % (HOST,), headers=grafana_headers(api_key))
+        if r.status_code == httplib.OK:
+            print '   * Removing DBaaS dashboard'
+            r = requests.delete('%s/api/dashboards/uid/pmm-dbaas' % (HOST,), headers=grafana_headers(api_key))
+            if r.status_code != httplib.OK:
+                print r.status_code, r.content
+        return
+
+    print ' * DBaaS is enabled'
+    r = requests.get('%s/api/dashboards/uid/pmm-dbaas' % (HOST,), headers=grafana_headers(api_key))
+    if r.status_code != httplib.OK:
+        print '   * Adding DBaaS dashboard'
+        with open('/usr/share/percona-dashboards/pmm-app/dist/dashboards/pmm-dbaas.json', 'r') as dashboard_file:
+            dashboard = json.loads(dashboard_file.read())
+        data = json.dumps({'dashboard': dashboard})
+        r = requests.post('%s/api/dashboards/import' % (HOST,), data=data, headers=grafana_headers(api_key))
+        if r.status_code != httplib.OK:
+            print r.status_code, r.content
+    else:
+        print '   * DBaaS dashboard has already been added'
+
+
+def check_active_process(processname):
+    ps = os.popen("ps -Af").read()
+    proccount = ps.count(processname)
+
+    if proccount > 0:
+        print' * %s service is running' % (processname,)
+        return True
+
+
+def args_parser():
+   global GRAFANA_DB_DIR
+   global OLD_VERSION_FILE
```

Comment:
This looks like not good practice to use "global" in general and for constants specifically.
Could you please change this to local variables and return this from function.